cg/src/utils/jwt.js

```diff
@@ -42,7 +42,7 @@ exports.managementJWT = new JWT(
   'HS256',
   process.env.MANAGEMENT_SECRET,
   process.env.MANAGEMENT_SECRET,
-  { expiresIn: getTokenExpiry() }
+  { expiresIn: 5 }
```

Comment:
I think you intended this for testing purposes only and should be reverted back to `getTokenExpiry()`?

Reply:
Can we define an env file for testing or doesn’t make sense?

Reply:
I believe there is a default for token expiry, so nothing needs to be done for this setting

Reply:
i did not mean to change this. will revert